README.md

```diff
@@ -46,11 +187,13 @@ const conf = new loki.Config(BASE_URL);
 const client = new loki.Client(conf);
 
 export default () => {
-  client.push()
-  sleep(1)
+  client.push();
```

Comment:
Minor: In this example, I suggest using one or two of the other APIs which are more descriptive.